manifests/candlepin.pp

```diff
@@ -97,12 +96,6 @@
       command => "openssl pkcs12 -export -in ${tomcat_cert} -inkey ${tomcat_key} -out ${keystore} -name tomcat -CAfile ${ca_cert} -caname root -password \"file:${password_file}\" -passin \"file:${ca_key_password_file}\" ",
       creates => $keystore,
     } ~>
-    file { "/usr/share/${tomcat}/conf/keystore":
```

Comment:
This is a different file than https://github.com/Katello/puppet-katello/commit/4e1371e8c93d75465b1f5b5f8fd025b90c8b9424#diff-0a129cc5a540f220caade2da4fc3229bR55 ?

Reply:
The path is different, but since ```/usr/share/tomcat/conf``` points to ```/etc/tomcat``` this is actually the same file.

Reply:
So it is 

```
lrwxrwxrwx. 1 root tomcat 11 Apr 19 09:37 /usr/share/tomcat/conf -> /etc/tomcat